test/auxiliary/test_io.jl

```diff
@@ -216,10 +235,131 @@
     @test jo.freq == 10
     @test jo.fields == Dict(:a => [:damage], :b => default_fields_body,
-                            :c => default_fields_body)
+               :c => default_fields_body)
 
     o = Dict{Symbol,Any}(:path => "rootpath", :fields => Dict(:b => ("test", :damage)))
     @test_throws ArgumentError begin
         Peridynamics.get_job_options(ms, vv, o)
     end
 end
+
+@testitem "export_results Body" begin
+    temp_root = joinpath(@__DIR__, "temp_root_export_results_test")
+    rm(temp_root; recursive=true, force=true)
+
+    pos, vol = uniform_box(1, 1, 1, 0.5)
+    body = Body(BBMaterial(), pos, vol)
+    material!(body, horizon=1.5, E=1, rho=1, Gc=1)
+    vv = VelocityVerlet(steps=1)
+    dh = Peridynamics.threads_data_handler(body, vv, 1)
+    chunk = dh.chunks[1]
+
+    o = Dict{Symbol,Any}(:path => temp_root, :freq => 1)
+    options = Peridynamics.get_job_options(body, vv, o)
+
+    u_rand = rand(3, 8)
+    chunk.storage.displacement .= u_rand
+
+    Peridynamics.export_reference_results(dh, options)
+    n = 1
+    t = 0.0001
+    chunk_id = 1
+    n_chunks = 1
+    Peridynamics.export_results(dh, options, chunk_id, n, t)
+
+    pvtu_file_ref = joinpath(temp_root, "vtk", "timestep_000000.pvtu")
+    @test isfile(pvtu_file_ref)
+    pvtu_file_n = joinpath(temp_root, "vtk", "timestep_000001.pvtu")
+    @test isfile(pvtu_file_n)
+
+    r0 = read_vtk(pvtu_file_ref)
+    @test r0[:position] ≈ pos
+    @test r0[:displacement] ≈ u_rand
+    @test r0[:damage] == zeros(8)
+    @test r0[:time] == [0.0]
+
+    r = read_vtk(pvtu_file_n)
+    @test r[:position] ≈ pos
+    @test r[:displacement] ≈ u_rand
+    @test r[:damage] == zeros(8)
+    @test r[:time] == [t]
+
+    rm(temp_root; recursive=true, force=true)
+end
+
+@testitem "export_results MultibodySetup" begin
+    temp_root = joinpath(@__DIR__, "temp_root_export_results_test")
+    rm(temp_root; recursive=true, force=true)
+
+    pos_a, vol_a = uniform_box(1, 1, 1, 0.5)
+    body_a = Body(BBMaterial(), pos_a, vol_a)
+    material!(body_a, horizon=1.5, E=1, rho=1, Gc=1)
+
+    pos_b, vol_b = uniform_box(1, 1, 1, 0.5, center_z=1.5)
+    body_b = Body(BBMaterial(), pos_b, vol_b)
+    material!(body_b, horizon=1.5, E=1, rho=1, Gc=1)
+
+    ms = MultibodySetup(:a => body_a, :b => body_b)
+
+    vv = VelocityVerlet(steps=1)
+
+    dh = Peridynamics.threads_data_handler(ms, vv, 1)
+    dh_a = dh.body_dhs[1]
+    chunk_a = dh_a.chunks[1]
+    @test chunk_a.body_name === :a
+    dh_b = dh.body_dhs[2]
+    chunk_b = dh_b.chunks[1]
+    @test chunk_b.body_name === :b
+
+    o = Dict{Symbol,Any}(:path => temp_root, :freq => 1)
+    options = Peridynamics.get_job_options(ms, vv, o)
+
+    u_rand_a = rand(3, 8)
+    u_rand_b = rand(3, 8)
+    chunk_a.storage.displacement .= u_rand_a
+    chunk_b.storage.displacement .= u_rand_b
+
+    Peridynamics.export_reference_results(dh_a, options)
+    Peridynamics.export_reference_results(dh_b, options)
```

Comment:
Using `export_reference_results` on `dh` and not on `dh_a`/`dh_b` will increase the test coverage.